src/storages/types.ts

```diff
@@ -409,6 +409,7 @@ export interface ITelemetryRuntimeProducerSync {
   recordTokenRefreshes(): void;
   recordStreamingEvents(streamingEvent: StreamingEvent): void;
   recordSessionLength(ms: number): void;
+  recordUpdatesFromSSE(type: UpdatesFromSSEEnum, amount: number): void
```

Comment:
No need for the `amount` param. Increment by one, one per each streaming notification that doesn't force a fetch.